_posts/2024-11-19-release-notes-3.6.0.md

```diff
@@ -0,0 +1,207 @@
+---
+category: announcement
+permalink: /news/3.6.0
+title: "Scala 3.6.0 is now available!"
+---
+
+<!-- TODO: Headline image -->
+
+We're happy to announce the next minor release of Scala - 3.6.0 is finally out!
+
+# What’s new in 3.6.0?
+
+Besides multiple bugfixes, this release stabilises multiple experimental features introduced to Scala language after careful review and acceptance by the [Scala Improvement Proposal's Commitee](https://docs.scala-lang.org/sips/). Many of these changes can have a significant impact on the Scala syntax and are introducing new amazing possibilities in writing concise, typesafe as well as easier, and easier to maintain code.
+
+## SIP-47 - Clause Interleaving
```

Comment:
@sjrd Can I ask you for a technical review or some improvements, as you're one of the SIP authors